### ISSUE TITLE
Incorrect character extraction, CID string returned

### ISSUE THREAD

**Issue by aoking** [pdfminer/pdfminer.six#934]
This PDF file does not allow for correct string extraction.

[bs104761.pdf](https://github.com/pdfminer/pdfminer.six/files/13873023/bs104761.pdf)

```
$ python tools/pdf2txt.py bs104761.pdf | head
WARNING:pdfminer.pdfpage:The PDF <_io.BufferedReader name='bs104761.pdf'> contains a metadata field indicating that it should not allow text extraction. Ignoring this field and proceeding. Use the check_extractable if you want to raise an error in this case
(cid:2446)(cid:2040)(cid:633)(cid:926)(cid:999)(cid:977)(cid:925)(cid:962)(cid:959)(cid:939)(cid:949)(cid:1490)(cid:2268)(cid:1393)(cid:2302)

(cid:2879) (cid:2310) (cid:2864) (cid:2480) (cid:3503)

(cid:4009) (cid:4072) (cid:250) (cid:3301) (cid:250) (cid:1860) (cid:250)(cid:248) (cid:3284) (cid:1905) (cid:2127)

(cid:2885)(cid:3503)(cid:2304)(cid:231)(cid:2676)(cid:2706)(cid:633)(cid:2399)

(cid:1354)(cid:633)(cid:633)(cid:633)(cid:633)(cid:633)(cid:3816)
(cid:2184)

```

**Comment by pietermarsman** [comment]
I can replicate this.

**Comment by pietermarsman** [comment]
Closed by #935
